test/unit/specs/components/v-logo-loader.spec.js

```diff
@@ -0,0 +1,28 @@
+import { render, screen } from '@testing-library/vue'
+import VLogoLoader from '~/components/VLogoLoader/VLogoLoader.vue'
+
+jest.mock('~/utils/warn', () => ({
+  warn: jest.fn(),
+}))
+
+jest.mock('~/composables/use-media-query', () => ({
+  useReducedMotion: jest.fn(() => true),
+}))
+
+describe('VLogoLoader', () => {
+  it('should render the logo', () => {
+    render(VLogoLoader)
+    const element = screen.getByTestId('logo-loader')
+    expect(element).toBeInTheDocument()
+  })
+
+  describe('accessibility', () => {
+    it('should render differently when the user prefers reduced motion', () => {
+      render(VLogoLoader, {
+        props: { status: 'loading' },
+      })
+      const element = screen.getByTestId('logo-loader')
+      expect(element).toHaveAttribute('data-prefers-reduced-motion', 'true')
+    })
+  })
```

Comment:
Should we also add a complimentary test for when `useReducedMotion` returns `false`? Currently all the test cases have it returning the default mock implementation you gave of `true`.

Reply:
Added, great suggestion.